index.html

```diff
@@ -1,116 +1,176 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="Ru">
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
 </head>
+  
 <body>
-  <header class="main-header">
-      <nav class="main-navigation">
-          <a class="main-header-logo">
-          <img src="img/index-logo.svg" width="368" height="153" alt="Barbershop Borodinsky Logo">
-          </a>
-          <ul class="site-navigation">
-              <li><a href="info.html">Информация</a></li>
-              <li><a href="news.html">Новости</a></li>
-              <li><a href="price.html">Прайс-лист</a></li>
-              <li><a href="contacts.html">Контакты</a></li>
-          </ul>
-          <ul class="user-navigation">
-              <li class="login-link">Вх од</li>
-          </ul>
-      </nav>
-  </header>
-  <main class="container">
+ <header class="header">
+    <nav class="navigation">
+        <a class="header-logo">
+            <img src="img/index-logo.svg" width="368" height="153" alt="Barbershop Borodinsky Logo">
+        </a>
+        
+        <ul class="site-navigation">
+            <li>
+                <a href="info.html">Информация
+                </a>
+            </li>
+            <li>
+                <a href="news.html">Новости
+                </a>
+            </li>
+            <li>
+                <a href="price.html">Прайс-лист
+                </a>
+            </li>
+            <li>
+                <a href="contacts.html">Контакты
+                </a>
+            </li>
+        </ul>
+   
+        <ul class="user-navigation">
+              <li class="login-link">Вх_од
+              </li>
+        </ul>
+    </nav>
 
-<h1 class="visually-hidden">Барбершоп "Бородинский"</h1>
+ </header>
+    <main class="container">
```

Comment:
отступы у мейна много
```suggestion
<main class="container">
```